File: src/main/java/org/example/bankramenserver/domain/recurring/exception/InvalidRecurringPaymentTransactionException.java
package org.example.bankramenserver.domain.recurring.exception;

import org.example.bankramenserver.global.error.exception.ErrorCode;
import org.example.bankramenserver.global.error.exception.GlobalException;

public class InvalidRecurringPaymentTransactionException extends GlobalException {

    public InvalidRecurringPaymentTransactionException() {
        super(ErrorCode.INVALID_RECURRING_PAYMENT_TRANSACTION);
    }
}

File: src/main/kotlin/org/example/bankramenserver/domain/recurring/exception/InvalidRecurringPaymentTransactionException.kt
package org.example.bankramenserver.domain.recurring.exception

import org.example.bankramenserver.global.error.exception.ErrorCode
import org.example.bankramenserver.global.error.exception.GlobalException

class InvalidRecurringPaymentTransactionException : GlobalException(ErrorCode.INVALID_RECURRING_PAYMENT_TRANSACTION)
